framework/sandstone_p.h

```diff
@@ -332,9 +332,15 @@ struct SandstoneApplication : public InterruptMonitor, public test_the_test_data
         child_exec_each_test,       // when parent is exec_each_test
     };
 
+    struct TestPlans {
```

Comment:
This is not a good name, I believe. It would be much easier to understand if it was called `TestSlice` and `Slice`. At least it would be consistent in terms of terminology.

Reply:
The plan is how you slice. I'll keep "Plan" in this struct but rename the inner fields.